src/notes/SingleHashtagKeywordStructuredFieldPlugin.jsx

```diff
@@ -12,6 +12,9 @@ function SingleHashtagKeywordStructuredFieldPlugin(opts) {
     const insertStructuredFieldTransform = opts.insertStructuredFieldTransform;
     const structuredFieldMapManager = opts.structuredFieldMapManager;
     function onBeforeInput(e, data, editorState) {
+        // short circuit if character typed is not a space
+        if (e.data !== ' ') return;
```

Comment:
(Not that we have even tested voice in a _minute_, butttt) This might brick voice (again, in case we haven't already bricked it with our many other changes)

Reply:
Hmm ok will remove it... was unsure if I wanted to include the short circuit in this PR.